CHANGELOG.next.asciidoc

```diff
@@ -798,6 +798,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Mark m365 defender, defender atp, okta and google workspace modules as GA {pull}23113[23113]
 - Added `alternative_host` option to google pubsub input {pull}23215[23215]
 - Support X-Forwarder-For in IIS logs. {pull}19142[192142]
+- Improve support of some logging options in PostgreSQL module. {pull}24607[24607]
```

Comment:
Maybe I'll add a more descriptive comment here like the PR title 😅

Reply:
Good point, comment improved. Thanks!